src/Utilities/BlazeExceptions.hpp

```diff
@@ -1,6 +1,8 @@
 // Distributed under the MIT License.
```

Comment:
Code looks good! Sorry I missed this last time, but could you add the hpp to the CMake lists?

Reply:
Fixed 👍